client/blocks/eligibility-warnings/hold-list.jsx

```diff
@@ -0,0 +1,129 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+import { map } from 'lodash';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import SectionHeader from 'components/section-header';
+
+// Mapping eligibility holds to messages that will be shown to the user
+// TODO: update supportUrls and maybe create similar mapping for warnings
+function getHoldMessages( translate ) {
+	return {
+		PLACEHOLDER: {
+			title: '',
+			description: '',
+			supportUrl: '',
+		},
+		TRANSFER_ALREADY_EXISTS: {
+			title: translate( 'Installation in progress' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **31** times:
`translate( 'Installation', { context: 'Navigation item'} )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).